client/src/pages/Listings/SitterCard/SitterCard.tsx

```diff
@@ -0,0 +1,80 @@
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Avatar from '@material-ui/core/Avatar';
+import Rating from '@mui/material/Rating';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import { Sitter } from '../../Profile/ProfileDetail/sampleData';
+import { Link } from 'react-router-dom';
+
+interface Props {
+  sitter: Sitter;
+}
+
+const SitterCard = ({ sitter }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Grid item xs={12} sm={5} md={4}>
+      <Box
+        width="80%"
+        display="flex"
+        justifyContent="space-evenly"
+        alignItems="center"
+        className={classes.sitterOuterBox}
+      >
+        <Paper elevation={6}>
```

Comment:
Optional: Why don't we consider the usage of the `Card` API here? (as they display content and actions on a single topic) 😄

Reply:
Applied. Replaced `Paper` with `Card` :)